### PR TITLE
correct some words

### PR DESCRIPTION
**Pull request by liliVicky**
Signed-off-by: liwenqi <vikilwq@zju.edu.cn>

### CODE REVIEW

File: contributors/design-proposals/access.md [hunk 1]
```diff
@@ -37,7 +37,7 @@ application or misconfiguration of K8s.
   - K8s API Path - attack by sending network messages to any K8s API endpoint.
   - Insider Path - attack on K8s system components. Attacker may have
 privileged access to networks, machines or K8s software and data. Software
-errors in K8s system components and administrator error are some types of threat
+errors in K8s system components and administrator errors are some types of threat
```
Comment (jonboulle) [inline contributors/design-proposals/access.md]: "administrator error" is appropriate (c.f. ["human error"](http://www.dictionary.com/browse/human-error))

File: contributors/design-proposals/clustering/README.md [hunk 2]
```diff
@@ -20,7 +20,7 @@ make docker
 
 The first run will be slow but things should be fast after that.
 
-To clean up the docker containers that are created (and other cruft that is left
+To clean up the docker containers that are created (and other crufts that is left
```
Comment (jonboulle) [inline contributors/design-proposals/clustering/README.md]: cruft is already a mass noun, crufts is not correct

File: contributors/design-proposals/cluster-deployment.md [hunk 3]
```diff
@@ -125,7 +125,7 @@ network solutions.
 
 1. flannel daemon will run as a standalone binary (not in docker container)
 2. flannel server will assign CIDRs to nodes outside of kubernetes; this will require restarting kubelet
-after reconfiguring network bridge on local machine; this will also require running master nad node differently
+after reconfiguring network bridge on local machine; this will also require running master node differently
```
Comment (jonboulle) [inline contributors/design-proposals/cluster-deployment.md]: this is trying to distinguish between the master and other nodes, "master node" is not correct

File: contributors/design-proposals/access.md (hunk 1 above)
Comment (liliVicky) [inline contributors/design-proposals/access.md]: I have modified. @jonboulle

**Comment by jonboulle** [comment]
lgtm

**Pull request merged**